apps/builder/app/builder/features/inspector/inspector.tsx

```diff
@@ -71,39 +62,41 @@ export const Inspector = ({ publish, navigatorLayout }: InspectorProps) => {
       skipDelayDuration={0}
     >
       <FloatingPanelProvider container={tabsRef}>
-        <Flex as={Tabs} grow ref={tabsRef} value={tab} onValueChange={setTab}>
-          <TabsList>
-            <TabsTrigger
-              value="style"
-              disabled={selectedInstance.component === "Slot"}
-            >
-              <DeprecatedText2>Style</DeprecatedText2>
-            </TabsTrigger>
-            {/* @note: events would be part of props */}
-            <TabsTrigger
-              value="props"
-              disabled={selectedInstance.component === "Slot"}
-            >
-              <DeprecatedText2>Properties</DeprecatedText2>
-            </TabsTrigger>
-            <TabsTrigger value="settings">
-              <DeprecatedText2>Settings</DeprecatedText2>
-            </TabsTrigger>
-          </TabsList>
-          <TabsContent value="style" css={contentStyle}>
-            <StylePanel publish={publish} selectedInstance={selectedInstance} />
-          </TabsContent>
-          <TabsContent value="props" css={contentStyle}>
-            <PropsPanelContainer
-              publish={publish}
-              key={selectedInstance.id /* Re-render when instance changes */}
-              selectedInstance={selectedInstance}
-            />
-          </TabsContent>
-          <TabsContent value="settings" css={contentStyle}>
-            <SettingsPanel />
-          </TabsContent>
-        </Flex>
+        <PanelTabs
+          ref={tabsRef}
+          value={selectedInstance?.component === "Slot" ? "settings" : tab}
+          onValueChange={setTab}
+          asChild
+        >
+          <Flex direction="column">
+            <PanelTabsList>
+              {selectedInstance.component !== "Slot" && (
+                <>
+                  <PanelTabsTrigger value="style">Style</PanelTabsTrigger>
+                  {/* @note: events would be part of props */}
+                  <PanelTabsTrigger value="props">Properties</PanelTabsTrigger>
+                </>
+              )}
```

Comment:
We used to disable these when a Slot is selected. But now I hide them because trigger doesn't have a disabled state

<img width="338" alt="Screenshot 2023-05-18 at 13 41 26" src="https://github.com/webstudio-is/webstudio-builder/assets/825702/8c1163b3-bb57-4bc4-88b6-1ea92dcea880">

Reply:
Good, hiding is better since the difference between disabled vs inactive might not be obvious